recipes/zeus_expected/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.cmake import cmake_layout
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+from conan.tools.files import apply_conandata_patches, export_conandata_patches
+import os
+
+required_conan_version = ">=1.54.0"
+
+
+class ZeusExpectedConan(ConanFile):
+    name = "zeus_expected"
+
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zeus-cpp/expected"
+    description = "Backporting std::expected to C++17."
+    topics = ("cpp17", "expected")
+    license = "MIT"
+
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    # patches applied
+    # no_copy_source = True
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "gcc": "7",
+            "clang": "5",
+            "Visual Studio": "15.7",
+            "apple-clang": "10",
+        }
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if min_version and Version(self.settings.compiler.version) < min_version:
+            raise ConanInvalidConfiguration(
+                f"{self.name} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
```

Comment:
```suggestion
        basic_layout(self, src_folder="src")
```

Cmake layout is nice when you are actually using CMake. The layout organizes the folder tree used when building and copying sources. But in this case, you want only to copy files, so basic_layout is lighter and fits better.